plugins/musicbrainzcovers/__init__.py

```diff
@@ -16,15 +16,19 @@
 import logging
 import urllib2
 
-from xl import common, covers, providers
+from xl import common, covers, main, providers
 
 import musicbrainzngs
 
 logger = logging.getLogger(__name__)
 
-musicbrainzngs.set_useragent(
-    'Exaile_MusicBrainz_Covers', '1.0.0', 'https://exaile.org/'
-)
+
+def __init_musicbrainzngs():
+    version = main.exaile().get_user_agent_for_musicbrainz()
```

Comment:
I don't think this exists yet? Also, I think the function should stay inside the plugin.

Reply:
You're right about it not existing yet. Looks like I am confused ;)